### PR TITLE
[EOS-22958] User msg should be displayed for exceptions

### PR DESCRIPTION
**Pull request by mazinamdar**
Signed-off-by: mazinamdar <mazhar.inamdar@seagate.com>

### CODE REVIEW

**Review by divya-vijay — APPROVED** [review]
Looks good

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by SomwanshiAnjali — APPROVED** [review]

File: lr-cli/cortx_setup/main.py [hunk 1]
```diff
@@ -63,11 +63,15 @@ def main():
     args = vars(args)
     cls_name = args.pop('command')
     cls = getattr(commands, cls_name)
-    res = cls().run(**args)
-    if res:
-        res = json.dumps(res)
-        Log.logger.debug(f"CLI out {res}")
-        print(res)
+    try:
+        res = cls().run(**args)
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/main.py:67]: Lets use good names such as: `result`

File: lr-cli/cortx_setup/main.py [hunk 2]
```diff
@@ -63,11 +63,15 @@ def main():
...
+        if res:
+            res = json.dumps(res)
+            Log.logger.debug(f"CLI out {res}")
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/main.py:70]: Change log message:
`CLI output: {result}`

**Review by nitin-seagate — APPROVED** [review]
Looks good to me... Please address minor comments

File: lr-cli/cortx_setup/main.py (hunk 1 above)
Comment (mazinamdar) [reply lr-cli/cortx_setup/main.py]: Done

File: lr-cli/cortx_setup/main.py (hunk 2 above)
Comment (mazinamdar) [reply lr-cli/cortx_setup/main.py]: Done

**Pull request merged**